README.md

```diff
@@ -155,6 +155,149 @@ function createRewire(requiredParams){
 module.exports = createRewire;
 ```
 
+## Expanded Configuration Options
+By default, the `override-config.js` file exports a single function to use when customising the webpack configuration for compiling your react app in development or production mode. It is possible to instead export an object from this file that contains up to three fields, each of which is a function. This alternative form allows you to also customise the configuration used for Jest (in testing), and for the Webpack Dev Server itself.
+
+```javascript
+module.exports = {
```

Comment:
now we have 2 almost similar examples: here and in `4) Example object-style config-overrides.js` section

Reply:
Oops. I was working further down in the file, and had forgotten that I had already created an initial example. I've merged the details for the two into the first of the examples.